style.css

```diff
@@ -0,0 +1,10 @@
+body{
+    padding: 0;
+    margin: 0;
+  }
+  #main{
+    background: silver;
+    text-align: center;
+    
+  }
```

Comment:
Kindly follow the right CSS best practice to avoid linter errors

Reply:
Kindly be more specific and explain what kind of CSS practices the student should follow, what makes this code unclean, not good? 

[OPTIONAL] You should give a more important reason than "avoiding linter errors". It could be something like "your code looks unclean and hardly readable".